app/factories/shopify/variant_factory.rb

```diff
@@ -0,0 +1,31 @@
+module Shopify
+  class VariantFactory
+    def initialize(spree_variant, shopify_variant)
+      @spree_variant = spree_variant
+      @shopify_variant = shopify_variant
+    end
+
+    def perform
+      shopify_variant.weight = spree_variant.weight
+      shopify_variant.weight_unit = 'oz'
+      shopify_variant.price = spree_variant.price
+      shopify_variant.sku = spree_variant.sku
+      shopify_variant.updated_at = spree_variant.updated_at
+      generate_options!
+
+      shopify_variant
+    end
+
+    private
+
+    attr_accessor :spree_variant, :shopify_variant
+
+    def generate_options!
+      shopify_variant.option1 = spree_variant.sku
```

Comment:
I take your point. But, imagine yourself as a new collaborator on the project in six months, or an end-user of the integration in a year. What would make you feel like this idiosyncrasy is well-documented?